css/styles.css

```
@@ -13398,1507 +13398,539 @@ th {
-/* STAK-217: Card hover lift */
-.retail-price-card:hover {
-  transform: translateY(-2px);
-  box-shadow: 0 8px 24px rgba(0, 0, 0, 0.22);
-  border-color: var(--primary, #3b82f6);
+
+.empty-state-icon {
+  width: 48px;
+  height: 48px;
+  margin-bottom: 1rem;
+  color: var(--border-hover);
+  fill: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)